### PR TITLE
Add Twitter, Discord, Mastodon, and Matrix social links

### PR DESCRIPTION
**Pull request by GreenXenith**

### CODE REVIEW

**Comment by celeron55** [comment]
So, I think we need to officialize open alternatives at the same time.

For Discord it would be Matrix. It's not clear what the room or community should be, but the current link to the current room is https://matrix.to/#/+minetest:tchncs.de

For Twitter, I guess it has to be Mastodon. Do we have anything on Mastodon? If not, does someone want to set that up?

**Comment by rubenwardy** [comment]
Yes, I run this account on mastodon: http://fosstodon.org/minetest

**Comment by GreenXenith** [comment]
I had intended to include Mastodon but for some reason I couldn't find the official user at the time.

Mastodon and Matrix have been added. I also split the community list into community and social mainly for size reasons (loosely sorted by activity). Feel free to suggest a different categorization.

**Review by celeron55 — CHANGES_REQUESTED** [review]
I'm ok with the content, altough what about listing the non-proprietary ones on top? I don't want it to feel like we're promoting Reddit, Twitter and Discord. Looks-wise I don't care.

**Comment by GreenXenith** [comment]
Reordered socials to promote FOSS platforms. Order based mostly on color.
image
Footer list has been reordered to reflect changes.

**Review by celeron55 — APPROVED** [review]
Looks good in the screenshot. Can I request someone to look through the code and merge if it's reasonable? I'm not exactly familiar with Jekyll.

File: _includes/footer.html [hunk 1]
```diff
@@ -23,18 +23,28 @@ <h5 class="footer-title">Content</h5>
...
           <li><a href="{{ '/servers/' | relative_url }}">Servers</a></li>
           <li><a href="https://wiki.minetest.net">Player Wiki</a></li>
+        </ul>
+      </div>
+
+      <div class="column is-6-mobile">
+        <h5 class="footer-title">Social</h5>
+        <ul class="list-unstyled">
+          <li><a href="https://matrix.to/#/+minetest:tchncs.de">Matrix</a></li>
+          <li><a href="https://fosstodon.org/@minetest">Mastodon</a></li>
```
Comment (rubenwardy) [inline _includes/footer.html:40]:
```suggestion
          <li><a href="https://fosstodon.org/@minetest" rel="me">Mastodon</a></li>
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**